src/Security/SecurityControllerBuilder.php

```diff
@@ -55,4 +63,19 @@ public function addLoginMethod(ClassSourceManipulator $manipulator)
         );
         $manipulator->addMethodBuilder($loginMethodBuilder);
     }
+
+    public function addLogoutMethod(ClassSourceManipulator $manipulator)
+    {
+        $logoutMethodBuilder = $manipulator->createMethodBuilder('logout', null, false, ['@Route("/logout", name="app_logout" , methods={"GET"})']);
```

Comment:
`methods=""` part is not needed. In fact, I think the `LogoutListener` will allow any HTTP method, so better to stay consistent with that.